src/main/kotlin/no/nav/syfo/oppfolgingstilfelle/person/api/OppfolgingstilfelleArbeidstakerApi.kt

```diff
@@ -0,0 +1,35 @@
+package no.nav.syfo.oppfolgingstilfelle.person.api
+
+import io.ktor.server.application.*
+import io.ktor.server.response.*
+import io.ktor.server.routing.*
+import no.nav.syfo.application.api.access.APIConsumerAccessService
+import no.nav.syfo.application.api.personIdent
+import no.nav.syfo.oppfolgingstilfelle.OppfolgingstilfelleService
+import no.nav.syfo.oppfolgingstilfelle.person.domain.toOppfolgingstilfelleDTOList
+import no.nav.syfo.util.*
+
+const val oppfolgingstilfelleArbeidstakerApiV1Path = "/api/v1/arbeidstaker/oppfolgingstilfelle"
```

Comment:
Burde kanskje vært i flertall, men de andre api'ene er ikke det - så best med entall for intern konsistens i app'en.